cmd/podman/cliconfig/config.go

```diff
@@ -117,14 +117,15 @@ type DiffValues struct {
 
 type ExecValues struct {
 	PodmanCommand
-	Env          []string
-	Privileged   bool
-	Interfactive bool
-	Tty          bool
-	User         string
-	Latest       bool
-	Workdir      string
-	PreserveFDs  int
+	DetachKeys  string
```

Comment:
I wonder if we shouldn't store the ones used during create with the container, and default to them for other attach sessions... Not necessary for this PR though.